### PR TITLE
HAWQ-1319. Expand pom.xml file for src/test directory structure.

### PR DESCRIPTION
**Pull request by edespino**
This has two goals:
* Catch missing ASF header from java files.
* Identify archive (jar) files. NOTE: If found, this does not cause
  RAT to fail.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -129,7 +129,51 @@
               <exclude>src/pl/**/*</exclude>
               <exclude>src/port/*</exclude>
               <exclude>src/template/*</exclude>
-              <exclude>src/test/**/*</exclude>
+              <exclude>src/test/**/*.c</exclude>
```
Comment (paul-guo-) [inline pom.xml]: I'm wondering whether we should not exclude some of the text files (source files, makefiles, etc) which we could add header files.  I remember we have a auto-script for header adding. @linwen

**Comment by huor** [comment]
@edespino, the source file in feature suite should be included in pom. Please create another jira to add the license header for the source file once the pom is updated.

**Comment by huor** [comment]
@edespino, please close this pr and we will update pom and add license header in source file in https://issues.apache.org/jira/browse/HAWQ-1323

**Comment by edespino** [comment]
Closing this PR as @huor and team will be updating the pom.xml.  They will be adding ASF headers as appropriate to additional source files.

-=e

**Pull request closed without merging**